src/disko/image_management/image_view.py

```diff
@@ -0,0 +1,134 @@
+        cluster_combobox.pack()
+
+        # Add confirm button to confirm cluster selection
+        confirm_button = ttk.Button(cluster_selection_window, text="Confirm", command=lambda: self.confirm_cluster_selection(cluster_var.get(), cluster_selection_window))
+        confirm_button.pack()
+
+    def confirm_cluster_selection(self, selected_cluster, cluster_selection_window):
+        # Confirm cluster selection
+        if selected_cluster:
+            cluster_selection_window.destroy()  # Close the cluster selection window
+            self.image_collector.collect_images(selected_cluster)  # Collect images for the selected cluster
+            self.selected_cluster = selected_cluster  # Update selected cluster variable
+            self.display_image_data(selected_cluster)  # Display image data for the selected cluster
+        else:
+            messagebox.showerror("Error", "Please select a cluster.")  # Show error message if no cluster selected
+
+    def create_images_table_screen(self):
+        # Create a new window for displaying images table
+        images_table_window = tk.Toplevel(self.root)
+        images_table_window.title("Images Table")
+
+        # Add frame for layout
+        frame = ttk.Frame(images_table_window, style='DarkFrame.TFrame')
+        frame.pack(padx=20, pady=20)
+
+        # Create treeview for displaying image data
+        self.treeview = ttk.Treeview(frame, columns=['Image', 'TimeStamp'], show='headings', style='Custom.Treeview')
+        self.treeview.heading('Image', text='Image')  # Set column heading for image
+        self.treeview.heading('TimeStamp', text='TimeStamp')  # Set column heading for timestamp
+        self.treeview.pack(side='left', fill='both', expand=True)
+
+        # Add scrollbar to the treeview
+        scrollbar = ttk.Scrollbar(frame, orient='vertical', command=self.treeview.yview)
+        scrollbar.pack(side='right', fill='y')
+        self.treeview.configure(yscrollcommand=scrollbar.set)
+
+        # Retrieve image data for the selected cluster
+        if self.selected_cluster:
+            image_data = self.db.select_all('images')
+
+            # Populate the treeview with image data
+            for image in image_data:
+                self.treeview.insert('', 'end', values=(image[1], image[2]))  # Insert image data into the treeview
+        else:
+            messagebox.showerror("Error", "No cluster selected.")  # Show error message if no cluster is selected
+
+    def update_columns(self):
+        # Update column display based on checkbox selection
+        selected_columns = [column.get() for column in self.checkboxes.values()]
+        for col in self.columns:
+            if col in selected_columns:
+                self.treeview.column(col, display=True)
+            else:
+                self.treeview.column(col, display=False)
+    
+   
+    def run(self):
```

Comment:
This function should get all the required data from the controller which it I turn should run it.
The main python script will run a function in the controller which will decide how to output the data.